app.js

```diff
@@ -0,0 +1,123 @@
+
+        this.render();
+    }
+
+    addTask = (e, task = this.inputAddTask.value, id = Date.now()) => {
+        e.preventDefault();
+
+        if (task === '') return alert('cos chcialbys zapamietac, nie puste znaki...');
+        let taskItem = {
+            id: id,
+            task: task,
+            completed: false,
+        }
+
+        this.toDoTasks.push(taskItem);
+        this.inputAddTask.value = '';
+        this.render();
+    }
+
+    removeTask = (e, taskList = this.toDoTasks) => {
+        const idOfTaskToDelete = e.target.parentNode.dataset.key;
+
+        for (let i = 0; i < taskList.length; i++) {
+            if (taskList[i].id == idOfTaskToDelete) {
+                taskList.splice(i, 1);
+                break;
+            }
+        }
+
+        this.render();
+    }
+
+    changeStatus = (e, taskList = this.toDoTasks) => {
+        const idOfTaskToChange = e.target.parentNode.dataset.key;
+
+        for (let i = 0; i < taskList.length; i++) {
+            if (taskList[i].id == idOfTaskToChange) {
+                taskList[i].completed = e.target.checked;
+                break;
+            }
+        }
+
+        this.render();
+    }
+
+    clearView = () => {
+        const allTasks = document.querySelectorAll('.task');
+        allTasks.forEach(el => {
+            el.remove();
+        })
+
+    }
+
+    render = (taskList = this.toDoTasks) => {
+        this.clearView();
+
+        for (let i = 0; i < taskList.length; i++) {
+
+            const taskContainer = document.createElement('div');
+            taskContainer.classList.add('task');
+            this.taskSection.appendChild(taskContainer);
+            taskContainer.dataset.key = taskList[i].id;
+
+            const contentOfTask = document.createElement('p');
+            contentOfTask.classList.add('taskText');
+            contentOfTask.textContent = taskList[i].task;
+            taskContainer.appendChild(contentOfTask);
+
+            const checkbox = document.createElement('input');
+            checkbox.type = 'checkbox';
+            checkbox.classList.add('checkbox');
+            checkbox.checked = taskList[i].completed;
+            taskContainer.appendChild(checkbox);
+
+            if (checkbox.checked) {
+                contentOfTask.classList.add('done');
+            }
+
+            const deleteButton = document.createElement('button');
+            deleteButton.classList.add('deleteBtn');
+            deleteButton.textContent = 'usuń';
+            taskContainer.appendChild(deleteButton);
+        }
+
+        const checkboxes = document.querySelectorAll('.checkbox');
+
+        checkboxes.forEach(el => {
+            el.addEventListener('change', this.changeStatus);
+        })
+
+        const deleteButtons = document.querySelectorAll('.deleteBtn');
+
+        deleteButtons.forEach(btn => {
+            btn.addEventListener('click', this.removeTask);
+        })
+    }
+
+}
+
+const toDo = new ToDoList();
```

Comment:
Rada na przyszłość - pamiętaj, żeby ZAWSZE zostawiać pustą linię na końcu pliku. To się tyczy każdego, JSa, HTMLa czy CSSów. 

Żeby zbytnio nie tłumaczyć chodzi o to, że przy minifikacji czasami pliki bez tej koncowej pustej linii się po prostu 'psują'. W skrócie.